### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by a-frantz**

### CODE REVIEW

**Comment by claymcleod** [comment]
Is this ready to be reviewed?

**Comment by a-frantz** [comment]
There's still the issue of the app not recognizing that large files have completed downloading. The progress bar hangs with 0-4 seconds estimated time remaining even though it's finished. Then there are two other small issues I'm not sure need to be addressed in this release. They aren't critical bugs.
https://trello.com/c/6FoMVvTY/189-issues-with-sjdta
That's the trello card where I've documented all the issues.

**Comment by claymcleod** [comment]
Looks pretty good, let's sit down and test together when you have built a release.

File: app/src/backend/queue.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const PRIORITY = {
 /**
  * @todo Better exposure of debugging.
  */
-const enableDebug = false;
+const enableDebug = true;
```
Comment (claymcleod) [inline app/src/backend/queue.ts]: Should this be false?

File: app/src/frontend/vue/substores/storeFiles.js [hunk 2]
```diff
@@ -114,22 +114,32 @@ export default function(ref) {
             elem.errored = true;
           }
           let process = null;
+          let cancelToken = null;
           if (state.currPath === 'upload') {
-            process = state.operationProcesses[elem.path];
+            process = state.operationUploadProcesses[elem.path];
+            if (process) {
+              console.log(process);
```
Comment (claymcleod) [inline app/src/frontend/vue/substores/storeFiles.js]: Should these console.logs be removed?

File: app/src/backend/queue.ts (hunk 1 above)
Comment (a-frantz) [reply app/src/backend/queue.ts]: O yes it should. They slipped in while I was committing

File: app/src/frontend/vue/substores/storeFiles.js (hunk 2 above)
Comment (a-frantz) [reply app/src/frontend/vue/substores/storeFiles.js]: Yes looks like I was sloppy with my commits. I'll take another look through make sure nothing else I used for debugging slipped through.

File: app/src/backend/dx.ts [hunk 3]
```diff
@@ -186,47 +187,45 @@ export const downloadDxFile = async (
   _fileRawSize: number,
   downloadLocation: string,
   updateCb: ResultCallback<number>,
-  finishedCb: SuccessCallback<request.Response>,
-): Promise<request.Request | null> => {
+  finishedCb: SuccessCallback<any>,
+): Promise<any | null> => {
```
Comment (jsunny23) [inline app/src/backend/dx.ts]: It would be good to add Axios type definitions.

**Pull request merged**